docs/source/migration-engine.md

```diff
@@ -24,9 +22,9 @@ server.listen().then(({ url }) => {
 });
 ```
 
-## Starting Engine Proxy
+## Starting Engine Proxy as a Sidecar
```

Comment:
I think we moved away from the "sidecar" terminology because it never made sense to anybody